src/storage.js

```diff
@@ -0,0 +1,42 @@
+const setToLocalStorage = (toDo) => {
+  localStorage.setItem('toDo', JSON.stringify(toDo));
+};
```

Comment:
To take advantage of the convenience of arrow functions, you can write:
`const setToLocalStorage = toDo => localStorage.setItem('toDo', JSON.stringify(toDo));
`;